json-endpoints/jetpack/class.jetpack-json-api-endpoint.php

```diff
@@ -94,6 +94,11 @@ protected function validate_call( $_blog_id, $capability, $check_validation = tr
 	 * @return bool|WP_Error
 	 */
 	protected function check_capability( $capability ) {
+		// If this endpoint allows site based authentication (aka Jetpack blog token) skip
+		// capabilities check.
```

Comment:
If this endpoint allows site based authentication (aka Jetpack blog token) AND the current request is signed with a blog token...

Reply:
Since we changed the method name to `accepts_` I'd prefer to say if the endpoints accepts site based authentication and leave the internals explained where this method lives.